src/pybind/ceph_volume_client.py

```diff
@@ -318,7 +317,7 @@ def recover(self):
 
         log.debug("Recovered from partial auth updates (if any).")
 
-    def _recover_auth_meta(auth_id, auth_meta):
+    def _recover_auth_meta(self, auth_id, auth_meta):
```

Comment:
Can we exclude this fix from the commit? Rest of the fixes in the commit seem to be coding style or py3 compat related. I think it'd be better to file a separate tracker for the broken recover phase. I also spotted other bugs with this part of the code. I can file the tracker and fix all the issues related to the recovery as a different PR.